docs/drive-management.md

```diff
@@ -0,0 +1,111 @@
+# Drive management
+
+## Prerequisites
+* Working DirectPV plugin. To install the plugin, refer to the [plugin installation guide](./installation.md#directpv-plugin-installation).
+* Working DirectPV CSI driver in Kubernetes. To install the driver, refer to the [driver installation guide](./installation.md#directpv-csi-driver-installation).
+
+## Add drives
+Drives are added to DirectPV to provision volumes. This involves a two step process as shown below.
+
+1. Run `discover` command - `discover` command probes eligible drives from DirectPV nodes and stores drive information in a YAML file. You should carefully examine the YAML file and set the `select` field to `yes` or `no` value to indicate drive selection. `select` field is set to `yes` value by default. Below is an example of the `discover` command:
+```sh
+# Probe and save drive information to drives.yaml file.
+$ kubectl directpv discover
+
+ Discovered node 'master' ✔
+ Discovered node 'node1' ✔
+
+┌─────────────────────┬────────┬───────┬─────────┬────────────┬──────┬───────────┬─────────────┐
+│ ID                  │ NODE   │ DRIVE │ SIZE    │ FILESYSTEM │ MAKE │ AVAILABLE │ DESCRIPTION │
+├─────────────────────┼────────┼───────┼─────────┼────────────┼──────┼───────────┼─────────────┤
+│ 252:16$ud8mwCjPT... │ master │ vdb   │ 512 MiB │ -          │ -    │ YES       │ -           │
+│ 252:16$gGz4UIuBj... │ node1  │ vdb   │ 512 MiB │ -          │ -    │ YES       │ -           │
+└─────────────────────┴────────┴───────┴─────────┴────────────┴──────┴───────────┴─────────────┘
+
+Generated 'drives.yaml' successfully.
+
+# Show generated drives.yaml file.
+$ cat drives.yaml
+version: v1
+nodes:
+    - name: master
+      drives:
+        - id: 252:16$ud8mwCjPTH8147TysmiQ2GGLpffqUht6bz7NtHqReJo=
+          name: vdb
+          size: 536870912
+          make: ""
+          select: "yes"
+    - name: node1
+      drives:
+        - id: 252:16$gGz4UIuBjQlO1KibOv7bZ+kEDk3UCeBneN/UJdqdQl4=
+          name: vdb
+          size: 536870912
+          make: ""
+          select: "yes"
+
+```
+
+2. Run `init` command.
+
+   The `init` command creates a request to add the selected drives in the YAML file generated using the `discover` command. As this process wipes out all data on the selected drives, wrong drive selection will lead to permanent data loss. Below is an example of `init` command:
+
+```sh
+$ kubectl directpv init drives.yaml
```

Comment:
```suggestion
$ kubectl directpv init drives.yaml --dangerous
```

Reply:
We should not expose `--dangerous` flag anywhere. A copy/paste such reference would lead to data loss.